corehq/apps/users/forms.py

```diff
@@ -537,8 +537,9 @@ def __init__(self, project, user, *args, **kwargs):
         email_string = u"@{}.commcarehq.org".format(project.name)
         max_chars_username = 80 - len(email_string)
         self.project = project
+        self.domain = self.project.name
```

Comment:
I think it was just that `self.domain` was available on the other form classes here, and I was mildly annoyed when it didn't work here.